src/main/java/org/sagebionetworks/migration/async/checksum/ChecksumRangeExecutor.java

```diff
@@ -80,8 +80,10 @@ public boolean hasNext() {
 			}
 			// Start n number of backup jobs for the mismatched ID range.
 			RangeChecksum misMatchRange = mismatchedRanges.next();
-			lastBackupJobs = backupJobExecutor.executeBackupJob(type, misMatchRange.getMinimumId(),
-					misMatchRange.getMaximumId());
+			// Fix for PLFM-6551, the bin numbers need to drive the backup range.
```

Comment:
If the min/max are not used anymore it might be worth deleting it from the API/migration client.